### PR TITLE
Blog fix - Rust Lifetimes: A Complete Guide to Ownership and Borrowing

### PR DESCRIPTION
**Pull request by msali123**
Fixes done.

### CODE REVIEW

**Review by adamgordonbell — APPROVED** [review]

File: blog/_posts/2023-07-24-rust-lifetimes-ownership-burrowing.md [hunk 1]
```diff
@@ -822,6 +795,6 @@ Although Rust's approach to memory management through ownership and borrowing an
 
 In this article, you learned all about the basics of Rust lifetimes, including borrowing and references, lifetime syntax, and annotations. You also explored advanced topics, such as lifetime subtyping, higher-ranked trait bounds, and associated types and lifetimes. All the code samples for this tutorial are available in [this GitHub repo](https://github.com/ECJ222/rust-lifetimes).
 
-With the information and examples provided here, you should have a solid understanding of Rust's lifetimes and their importance in writing safe and efficient code. If you want to keep learning, try exploring the [official Rust documentation and community resources](<https://www.rust-lang.org/lea>
+With the information and examples provided here, you should have a solid understanding of Rust’s lifetimes and their importance in writing safe and efficient code. If you want to keep learning, try exploring the [official Rust documentation and community resources](https://www.rust-lang.org/learn).
```
Comment (adamgordonbell) [inline blog/_posts/2023-07-24-rust-lifetimes-ownership-burrowing.md:798]:
```suggestion
With the information and examples provided here, you should have a solid understanding of Rust's lifetimes and their importance in writing safe and efficient code. If you want to keep learning, try exploring the [official Rust documentation and community resources](https://www.rust-lang.org/learn).
```

Comment (adamgordonbell) [reply blog/_posts/2023-07-24-rust-lifetimes-ownership-burrowing.md:798]: @msali123  I think this is causing the build failure.

Comment (msali123) [reply blog/_posts/2023-07-24-rust-lifetimes-ownership-burrowing.md]: Yes just pushed with the fix

**Comment by msali123** [comment]
@adamgordonbell [Preview link](https://64be807fb282a308010eaf4a--ephemeral-narwhal-3ac3b2.netlify.app/blog/rust-lifetimes-ownership-burrowing/)

**Pull request merged**